docs/src/semiimplicit/ExponentialRK.md

```diff
@@ -0,0 +1,58 @@
+```@meta
+CollapsedDocStrings = true
+```
+
+# OrdinaryDiffEqExponentialRK
+
+Methods for semi-linear differential equations.
+
+## Installation
+
+To be able to access the solvers in `OrdinaryDiffEqLinear`, you must first install them use the Julia package manager:
+
+```julia
+using Pkg
+Pkg.add("OrdinaryDiffEqExponentialRK")
+```
+
+This will only install the solvers listed at the bottom of this page.
+If you want to explore other solvers for your problem,
+you will need to install some of the other libraries listed in the navigation bar on the left.
+
+## Example usage
+
+```@eval
+first_steps = evalfile("./common_first_steps.jl")
+first_steps("OrdinaryDiffEqExponentialRK", "EPIRK5s3")
```

Comment:
No idea if the standard example is correct for this solver type.

Reply:
it is